target/linux/ipq40xx/base-files/etc/board.d/01_leds

```diff
@@ -36,6 +36,9 @@ engenius,eap1300)
 engenius,eap2200)
 	ucidef_set_led_netdev "lan1" "LAN1" "${boardname}:blue:lan1" "eth0"
 	ucidef_set_led_netdev "lan2" "LAN2" "${boardname}:blue:lan2" "eth1"
+	ucidef_set_led_wlan "wlan2g" "WLAN2G" "${boardname}:green:wlan2G" "phy0tpt"
+	ucidef_set_led_wlan "wlan5g" "WLAN5G" "${boardname}:green:wlan5G" "phy1tpt"
+	ucidef_set_led_netdev "lan" "LAN" "${boardname}:green:lan" "eth0"
```

Comment:
This seems broken, as the device identifier is missing.

Despite, please implement the WLAN LEDs via a trigger in the DTS files, just grep for "phy0tpt" in the DTS folder for a blueprint.